lib/index.js

```diff
@@ -3,12 +3,11 @@
  */
 'use strict';
 
-const {config: {constants}} = require('bedrock');
-const {v1: securityContextV1, v2: securityContextV2} =
-  require('security-context');
+const {config: {constants: bedrockConstants}} = require('bedrock');
+const {contexts, constants: contextConstants} = require('security-context');
 
-constants.SECURITY_CONTEXT_V1_URL = 'https://w3id.org/security/v1';
-constants.CONTEXTS[constants.SECURITY_CONTEXT_V1_URL] = securityContextV1;
-
-constants.SECURITY_CONTEXT_V2_URL = 'https://w3id.org/security/v2';
-constants.CONTEXTS[constants.SECURITY_CONTEXT_V2_URL] = securityContextV2;
+for(const c in contextConstants) {
+  const contextUrl = contextConstants[c];
+  bedrockConstants[c] = contextUrl;
+  bedrockConstants.CONTEXTS[contextUrl] = contexts.get(contextUrl);
+}
```

Comment:
I wonder if there is a better pattern for this sort of logic.  Some issues:
- It's going to be repeated in any bedrock-*-context package.
- If for some reason this package wasn't a peer dep everywhere, and two packages with different versions were used, this may cause incorrect initialization.  Putting in checks for such issues would be better in a bedrock common API call.
- This assumes a one to one mapping of every `constants` entry with `contexts` entries.  Two loops could address that.

Reply:
@davidlehn I think you're suggesting that we have a more robust `bedrock.util.initContext`, and I agree that would be a good idea.  I have created an issue/feature request here: https://github.com/digitalbazaar/bedrock/issues/40

@davidlehn do you want to wait until someone gets around to implementing a helper, or will you approve this as is?

Reply:
Would we need this new helper to be specific to contexts? Or would it apply to other similar modules with shared constants?

Reply:
@dlongley not sure.  I suggest that discussion about the proposed helper occur on the feature request thread: https://github.com/digitalbazaar/bedrock/issues/40

Reply:
this PR is now update to bedrock@2 which will not include jsonld.